jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -192,6 +196,9 @@ public boolean isEnqueued () {
 	 * @return	true if the Reference was successfully
 	 *			enqueued, false otherwise.
 	 */
+	/* [IF CRIU_SUPPORT] */
```

Comment:
This needs to be `/*[IF CRIU_SUPPORT]*/`, otherwise caused [JDK8 compilation failure](http://vmfarm.rtp.raleigh.ibm.com/job_output.php?id=65550217) 
```
/tmp/bld_55732/pConfig/pConfig_SIDECAR18-SE/src/java/lang/ref/Reference.java:119: error: cannot find symbol
	@NotCheckpointSafe
	 ^
  symbol:   class NotCheckpointSafe
```

FYI @TobiAjila

Reply:
Didn't see this before merge. @tajila If you don't fix this in the next few hours, I'll revert this for the long weekend.

Reply:
@JasonFengJ9 I dont understand the comment, `/*[IF CRIU_SUPPORT]*/,` was added

Reply:
And CRIU_SUPPORT is disabled for JDK8

Reply:
oh is it the spacing?

Reply:
https://github.com/eclipse-openj9/openj9/pull/17917/